xbmc/platform/darwin/DarwinUtils.mm

```diff
@@ -157,35 +165,24 @@
 {
   static bool ret = false;
   static std::once_flag flag;
-  std::call_once(flag, []
-  {
-    size_t path_size = 2*MAXPATHLEN;
-    char given_path[path_size];
-    memset(given_path, 0x0, path_size);
-    /* Get Application directory */
-    int result = GetExecutablePath(given_path, &path_size);
-    if (result == 0)
+  std::call_once(flag, [] {
+    auto executablePath = getExecutablePath();
+    auto sandboxPrefixPaths = {
+        // before iOS 8, probably can be removed entirely
+        @"/var/mobile/Applications/",
+        // since iOS 8
+        @"/var/mobile/Containers/Bundle/",
+        // since iOS 9? (iOS 9.3.5 already has this)
```

Comment:
On iOS 9.0.2 `/var/containers/` doesn't exist.

Reply:
Original comment for `/var/containers/Bundle/` was

> Some time after ios8, Apple decided to change this yet again

So iOS 9.0.2 uses `/var/mobile/Containers/Bundle/`?

Reply:
Yes, the apps installed from the store are located at `/var/mobile/Containers/Bundle/Application/`.

Reply:
thanks, will adjust the comment